app/services/invitation_manager.rb

```diff
@@ -27,6 +27,26 @@ def success?
 
   private
 
+    def find_role(role)
+      case role
+
+      when "Student"
```

Comment:
I'm wondering if we could refactor this with either a method call which access a hash that returns out the role to look up, or maybe just set an instance variable to the hash to access. Might be able to make it more dynamic and  slim down the lines of code?  @JStans12